liquibase-standard/src/test/java/liquibase/database/DatabaseListTest.java

```diff
@@ -43,4 +50,25 @@ public void databaseMatchesDbmsDefinition() {
         assertFalse(DatabaseList.definitionMatches("!h2,mysql", new H2Database(), false));
         assertTrue(DatabaseList.definitionMatches("!h2,mysql", new MySQLDatabase(), false));
     }
+
```

Comment:
Do you mind instead of adding this test here we create a new `DatabaseListTest` class into the database directory from the Groovy sources? Unless it's necessary or because of time constraints we tried to add tests in Groovy format, so if you want to migrate this one/both tests there that would be nice. Otherwise, I can do it.


In case you are not familiar with how to parameterize a test in Spock here's an [example](https://github.com/liquibase/liquibase/blob/4edcddc2d735cc08738a9781aeccfefde9627ee9/liquibase-standard/src/test/groovy/liquibase/changelog/DatabaseChangeLogTest.groovy#L787).

Reply:
Done. Are you okay with the test names?